### ISSUE TITLE
Warning while precompiling RLCore due to kwargs 

### ISSUE THREAD

**Issue by 3rdCore** [JuliaReinforcementLearning/ReinforcementLearning.jl#575]
It seems that some functions of `CircularArrayTrajectory` uses deprecated functions on kwargs : 

![image](https://user-images.githubusercontent.com/59280588/150154484-5f309aa0-a408-4fe9-a0fd-786df1ab4824.png)

Replacing : 
```julia
function CircularArrayTrajectory(; capacity, kwargs...)
    Trajectory(map(kwargs.data) do x
        CircularArrayBuffer{eltype(first(x))}(last(x)..., capacity)
    end)
end
```

by : 
```julia
function CircularArrayTrajectory(; capacity, kwargs...)
    Trajectory(map(values(kwargs)) do x
        CircularArrayBuffer{eltype(first(x))}(last(x)..., capacity)
    end)
end
```
would maybe fix this issue.

**Comment by findmyway** [comment]
I think it was fixed some time ago in #560 ? Which version are you using now? I'm not sure if I forgot to tag a new release...

https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/7adaf4a23aaee405cecc1476dd26082cc05364d7/src/ReinforcementLearningCore/src/policies/agents/trajectories/trajectory.jl#L56-L60

**Comment by findmyway** [comment]
A new version will be tagged soon. Thanks for reporting!
